### ISSUE TITLE
Getting admin password note wrong on Grafana chart

### ISSUE THREAD

**Issue by godber** [helm/charts#354]
After running the grafana chart it suggests I get the grafana admin password with the following text:

```
1. Get your 'admin' user password by running:

   printf $(printf '\%o' `kubectl get secret --namespace default lumpy-mandrill-grafana -o jsonpath="{.data.grafana-admin-password[*]}"`);echo
```

Running this, on OS X, results in the following output:

```
printf $(printf '\%o' `kubectl get secret --namespace default lumpy-mandrill-grafana -o jsonpath="{.data.grafana-admin-password[*]}"`);echo
error: error executing jsonpath "{.data.grafana-admin-password[*]}": string is not array or slice
-bash: printf: Error: invalid number
... snipped printf errors
-bash: printf: {}{"app":"lumpy-mandrill-grafana",: invalid number
gtkamrclchrnns
```

The first issue is with the `jsonpath`, I think it should be `-o jsonpath="{.data.grafana-admin-password}"` (removing the `[*]`).  Then the second issue is that `printf` doesn't seem to be achieving what needs to be done.  On OS X the following line will retrieve and decode the value:

```
kubectl get secret --namespace default lumpy-mandrill-grafana -o jsonpath="{.data.grafana-admin-password}" | base64 -D
```

This is a little more apparent if we decode the admin user:

```
kubectl get secret --namespace default lumpy-mandrill-grafana -o jsonpath="{.data.grafana-admin-user}" | base64 -D
```

So basically I think the following line needs to be changed:

https://github.com/kubernetes/charts/blob/master/stable/grafana/templates/NOTES.txt#L3

Also, I think there was another Chart that had this issue, but I can't remember what it was.

I can find and fix these if you confirm that I have properly identified the issue.

I have signed the CNCF CLA.

**Comment by viglesiasce** [comment]
This is also reported in #366 

@godber do you know if your solution works with a 1.4 client as well?

**Comment by godber** [comment]
At the moment no.  I appear to be running version 1.5.1:

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"5", GitVersion:"v1.5.1", GitCommit:"82450d03cb057bab0950214ef122b67c83fb11df", GitTreeState:"clean", BuildDate:"2016-12-22T13:56:59Z", GoVersion:"go1.7.4", Compiler:"gc", Platform:"darwin/amd64"}
```

**Comment by seanknox** [comment]
Closing due as this was filed awhile ago. @godber feel free to re-open if this is still an issue.
